packages/hoppscotch-sh-admin/src/components/users/Details.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+  <div class="rounded-md">
+    <div class="grid gap-6 mt-4">
+      <div v-if="user.photoURL" class="relative h-20 w-20">
+        <img class="object-cover rounded-3xl mb-3" :src="user.photoURL" />
+        <span
+          v-if="user.isAdmin"
+          class="absolute left-16 bottom-0 text-xs font-medium px-3 py-0.5 rounded-full bg-green-900 text-green-300"
+        >
+          {{ t('users.admin') }}
+        </span>
+      </div>
+
+      <div v-else class="bg-primaryDark w-16 p-3 rounded-2xl mb-3 relative">
+        <icon-lucide-user class="text-4xl" />
+        <span
+          v-if="user.isAdmin"
+          class="absolute left-16 bottom-0 text-xs font-medium px-3 py-0.5 rounded-full bg-green-900 text-green-300"
+        >
+          {{ t('users.admin') }}
+        </span>
+      </div>
```

Comment:
This can be unified. The dynamic things here are deciding whether to render an image or the icon based on the avatar URL availability and conditionally binding the classes to the parent `div`.